src/model/generateRestHandlers.ts

```diff
@@ -126,6 +126,7 @@ export function generateRestHandlers<
   baseUrl: string = '',
 ) {
   const primaryKey = findPrimaryKey(modelDefinition)!
+  const primaryKeyValue = findPrimaryKeyValue(modelDefinition)!
```

Comment:
We already know that `primaryKey` is the property name in `modelDefinition` that keeps the declaration of the primary key. We can access it directly instead of looking it up again in `findPrimaryKeyValue`:

```ts
const primaryKey = findPrimaryKey(modelDefinition)!
const primaryKeyValue = modelDefinition[primaryKey].getValue()
```

Reply:
This will show a TypeScript error related to type narrowing.
![CleanShot 2021-11-01 at 11 58 51](https://user-images.githubusercontent.com/7356098/139668108-16149f82-d4c2-4b18-9a89-ad2fdf5348dd.png)

Reply:
We can cast it's type like this:

```ts
const primaryKeyValue = (modelDefinition[primaryKey] as PrimaryKey).getValue()
```

Reply:
Changes pushed.